Gruntfile.coffee

```diff
@@ -542,6 +542,8 @@ module.exports = (grunt) ->
 
   # Register tasks
 
+  grunt.util.linefeed = '\n'
```

Comment:
What should be achieved by this line, and how is this related to the topic?
If there's a good reason, I can add it separately, but it should not be connected to this PR commit.

Reply:
I added it, cause I test in Windows. 
Without it I could not build